llama_stack/providers/inline/vector_io/sqlite_vec/sqlite_vec.py

```diff
@@ -729,7 +731,13 @@ def _delete():
 
         await asyncio.to_thread(_delete)
 
-    async def insert_chunks(self, vector_db_id: str, chunks: list[Chunk], ttl_seconds: int | None = None) -> None:
+    async def insert_chunks(
+        self,
+        vector_db_id: str,
+        chunks: list[Chunk],
+        ttl_seconds: int | None = None,
+        params: dict[str, Any] | None = None,
```

Comment:
Adding distance_metric in params instead of mentioning it explicitly - as not all providers need it. We can later expand params to include any other knobs for providers as they see fit.